### DISCUSSION TITLE
New YT ad-blocking counter-measures

### DISCUSSION THREAD

**Discussion by Kein** [TeamNewPipe/NewPipe#10988] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

https://9to5google.com/2024/04/15/youtube-app-block-ads/

### Additional information

> **Users will see a “The following content is not available on this app” error message or experience “buffering issues” when they try to play content though those alternative clients.**
> **[...]content throttling or other errors[...]**

So the question now is how do we distinguish between valid NP issues and YT counter-measures. Stuff like “buffering issues” is common, so is "other random errors"

**Comment by AudricV** [comment]
> - [x] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.

Duplicate of #10979. Please search (better, with different keywords?) for existing discussions and issues the next time you want to open a new discussion. Thank you.
